### PR TITLE
more Adherence calendar

### PR DESCRIPTION
**Pull request by emord**
changed spec for the adherence calendar. it no longer needs to be loading a case property in the session

buddy @czue

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js [hunk 1]
```diff
@@ -1061,6 +1061,26 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
                     !self.case_property() &&
                     !self.auto_select();
             });
+
+            self.validate = ko.computed(function () {
+                if (!self.config.caseConfigViewModel) {
+                    return;
+                }
+                if (!case_type) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js]: 'case_type' is not defined. (no-undef)

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js [hunk 2]
```diff
@@ -1061,6 +1061,26 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
...
+                    return "Case Type required";
+                }
+                if (case_tag) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js]: 'case_tag' is not defined. (no-undef)

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js [hunk 3]
```diff
@@ -1061,6 +1061,26 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
...
+                    if (!/^[a-zA-Z][\w_-]*(\/[a-zA-Z][\w_-]*)*$/.test(case_tag)) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js]: 'case_tag' is not defined. (no-undef)

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js [hunk 4]
```diff
@@ -1061,6 +1061,26 @@ hqDefine('app_manager/js/case-config-ui-advanced.js', function () {
...
+                        return "Case Tag: only letters, numbers, '-', and '_' allowed";
+                    }
+                    var tags = self.config.caseConfigViewModel.getCaseTags('all');
+                    if (_.where(tags, {value: case_tag}).length > 1) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js]: 'case_tag' is not defined. (no-undef)

**Comment by czue** [comment]
good looks

**Pull request merged**